texts/epn_paper.md

```diff
@@ -0,0 +1,71 @@
+# Let's talk online
+
+<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->
+
+Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home, hopefully safe and healthy.
```

Comment:
Your routine changed drastically over the course of the last couple of months due to the lockdowns following the novel coronavirus outbreak. You now mostly stay at home, hopefully safe and healthy.